src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,119 @@
 package co.inventorsoft.academy.collections.model;
 
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
+
 
 public class Range<T> implements Set<T> {
 
+    List<T> mylist  = new ArrayList();
+
+
+    public Range(float a, float b) {
+        if(!(a==b)){
+            while (a <=b) {
+                mylist.add((T)(Float) a);
+                a = a + 0.1f;
+            }}
+    }
+
+    public Range(int a, int b) {
+        if (!(a==b)){
```

Comment:
Create a generator and pass it as a parameter to of(T, T, Function<T, T>)

Reply:
corrected